helm_tests/other/test_keda.py

```diff
@@ -58,6 +58,17 @@ def test_keda_enabled(self, executor, is_created):
     @pytest.mark.parametrize(
         "executor", ["CeleryExecutor", "CeleryKubernetesExecutor", "CeleryExecutor,KubernetesExecutor"]
     )
+    def test_include_event_source_container_name_in_scaled_object(self, executor):
+        docs = render_chart(
+            values={
+                "workers": {"keda": {"enabled": True}, "persistence": {"enabled": False}},
+                "executor": executor,
+            },
+            show_only=["templates/workers/worker-kedaautoscaler.yaml"],
+        )
+        assert jmespath.search("spec.scaleTargetRef.envSourceContainerName", docs[0]) == "worker"
+
+    @pytest.mark.parametrize("executor", ["CeleryExecutor", "CeleryKubernetesExecutor"])
```

Comment:
Can you update this parametrization? The list of executors was updated recently :)

Reply:
Thanks! fixed it. 🤗